src/MEDS_DEV/tasks/criteria/abnormal_lab/in_icu/blood_chemistry/hyponatremia/first_24h.yaml

```diff
@@ -0,0 +1,48 @@
+description: >-
+  This task predicts whether the patient will have abnormal _sodium_ values within the next 24
+  hours, given the first 24 hours of their admission to the ICU. To be included, patients must have
+  at least one measurement for _sodium_ within the next 24 hours after their admission to ICU,
+  and their length of stay in ICU should be at least 24 hours.
+
+predicates:
+  icu_admission: ???
+  icu_discharge: ???
+  hospital_discharge: ???
+  death: ???
+  discharge_or_death:
+    expr: or(icu_discharge, death, hospital_discharge)
+
+  # Plain predicates
+  sodium:
+    code: ???
+  abnormally_low_sodium:
+    code: ???
+    value_min: null
+    value_max: ???
```

Comment:
Is this reference range different for men vs. women, or children vs. adults? Do we need more qualifiers and/or different ranges for different patients here?  Tagging @Gallifantjack for the clinical questions here.

Reply:
Same with the above.

Reply:
Hyponatremia has a standardized cutoff that applies equally to both men, women, and neonate/pediatric patients (< 135 mEq/L). As mentioned above, it's best to focus on adult patients (age > 18) for the tasks defined here for now.

Reply:
@mandiehyewon, I agree with the focus on adult patients -- but we should therefore encode that specifically in the inclusion criteria here, right? @justin13601, do we have any ability to filter by age or by duration of time of a window?

Reply:
How would age be represented in MEDS? Would it simply be subtracting the timestamp at a given event by the DOB? I wonder if one can also create a window in the task requiring an event as an adult (ie. 18 years after DOB or something like that)?

Reply:
At least for the MEDS_Extract_v0.0.7 version, one's birth date is represented as an event whose code is "MEDS_BIRTH" with a timestamp of the corresponding DOB, so we should calculate age dynamically for every triggered sample and filter it out if its age is less than 18 years by a window or something as @justin13601 mentioned.

Reply:
I think we can just do a time-derived window of the target age as @justin13601 suggests, and we can make it more elegant later. @Jwoo5, do you have a good sense of how to do that? We can just do a "no_children" window that ends at `trigger` and starts at `end - 18y` and has no `birth` events which are defined using the `MEDS_BIRTH` code. That should work, right?

Reply:
I just tested it for in-icu mortality task, but it turns out that there is no difference in the cohort numbers:
```yaml
  no_children:
    start: end - 6570d # 18 years = 365 * 18 days
    end: trigger
    start_inclusive: True
    end_inclusive: True
    has:
      birth: (None, 0)
```
and I checked that all the subjects from in-icu mortality (based on icu admission) and long-los (based on hosp admission) cohorts are of age > 18.
It seems like MEDS_Extract_v0.0.7 has already filtered out children patients, could you confirm it @mmcdermott?

Reply:
Sorry for the delayed confirmation, @mmcdermott. That's correct, we would like to specify cohort to be adults if we want to use this cutoff!